config/settings_schema.json

```diff
@@ -1397,6 +1366,41 @@
       }
     ]
   },
+  {
+    "name": "t:settings_schema.brand_information.name",
```

Comment:
Good point. I'm not 100% sure if we should add these defaults on this repo specifically (because of conflicts - did we do that for other settings recently?), but yes, ideally all our themes should have them. Also, I think we shouldn't have `"brand_description": "<p></p>"` as default, otherwise the setting will not be empty.

As for the order, it's not a big deal but makes it more organized. I would move them to above social links, yes :D